global.json

```diff
@@ -757,7 +757,7 @@
       "appTemplates": ["hello_world_mbaas_instance"]
     }, {
       "name": "OpenShift mBaaS Service",
-      "priority": 0.895,
```

Comment:
Shall we create a separate Jira Issue to discuss about the removal of this template? In the meantime I think we can merge this PR (if approved) as it doesn't change the order for that Service, it will remain in second position